templates/base.html

```diff
@@ -224,9 +223,10 @@
     </section>
 
     {% block js %}
-        <script src="https://cdn.ravenjs.com/3.17.0/raven.min.js" crossorigin="anonymous"></script>
-        <script>Raven.config('{% sentry_public_dsn "https" %}').install()</script>
-
+        <script
+            src='https://js.sentry-cdn.com/df3629be91944be79c790681d9a57f76.min.js'
+            crossorigin="anonymous">
+        </script>
```

Comment:
I think we need to activate the SDK outside of the 500-template (and with that, I don't think you would need to initialize it in the template), and I think we should add the integrity hash as well

Reply:
Also, why is this from `js.sentry-cdn.com` instead of `browser.sentry-cdn.com`? Also I don't think we need to download the SDK in the 500, if we download it in base